eth/tools/builder/chain/builders.py

```diff
@@ -257,7 +257,7 @@ def dao_fork_at(dao_fork_block_number: BlockNumber,
 
 @to_dict
 def _get_default_genesis_params(genesis_state: AccountState,
-                                ) -> Iterable[Tuple[str, Union[int, None, bytes, Address, Hash32]]]:
+                                ) -> Iterable[Tuple[str, Union[BlockNumber, int, None, bytes, Address, Hash32]]]:  # noqa: E501
```

Comment:
*nitpick*: I have a slight preference to reformat (e.g. put the parameter on the next line) instead of putting a `noqa: E501` on it. Especially if we consider that one day we'll be using black anyway.

Reply:
generally agreed.  I chose this after having done it the other way and finding the readability to be poor.